plugins/AtlasExplorer/io/nrrdloader/data/decoders/HexDecoder.cpp

```diff
@@ -31,22 +33,19 @@ class HexContentParser
 public:
     static std::string toBinary(std::string_view input)
     {
-        auto tokens = Tokenizer::fromView(input);
-
-        size_t expectedResultSize = 0;
-        for (auto token : tokens)
-        {
-            expectedResultSize += token.size();
-        }
+        auto tokenCount = brayns::StringCounter::countOneOf(input, brayns::StringConstants::tokenDelimiters);
+        auto expectedResultSize = input.length() - tokenCount;
 
         std::string result;
         result.reserve(expectedResultSize);
 
         std::string hexBuffer(3, '\0');
         size_t hexIndex = 0;
 
-        for (auto token : tokens)
+        while (!input.empty())
         {
+            auto token = brayns::StringExtractor::extractUntilOneOf(input, brayns::StringConstants::tokenDelimiters);
```

Comment:
extractToken ?

Reply:
extractToken extracts only space-separated tokens, whereas the nrrd files have more delimiters. And I dont think its correct to use the list of delimiters from a specific file format into a generic function that is opaque to the user, even more when there is API like extractUntilOneOf which does what its needed.